src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -576,7 +576,7 @@ private static function initPlaceholderFormatters()
                 return Helper::formatTime(time() - $bar->getStartTime());
             },
             'remaining' => function (ProgressBar $bar) {
-                if (!$bar->getMaxSteps()) {
+                if (null === $bar->getMaxSteps()) {
                     throw new \LogicException('Unable to display the remaining time if the maximum number of steps is not set.');
                 }
```

Comment:
Do you have a test for this change? A few lines below there's this:
```
$remaining = round((time() - $bar->getStartTime()) / $bar->getProgress() * ($bar->getMaxSteps() - $bar->getProgress()));

// 100 / 0 * (0 - 0)
```
This would result in a division by zero